### PR TITLE
Release 2021-11-05

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Comment by shawntanzk** [comment]
Release Notes:

New Terms:
bitufted dendrite cell morphology
retinal bipolar morphology

Edits: 
Edited definition of cystic, polycystic, monocystic
Changed orinates from to position originates from 
Rearranged bipolar/bitufted cell morphology

**Review by shawntanzk — APPROVED** [review]

**Comment by anitacaron** [comment]
Thanks, @shawntanzk!

File: src/ontology/reports/pato-edit.obo-obo-report.tsv [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Level	Rule Name	Subject	Property	Value
-WARN	missing_definition	part of	definition	
-WARN	missing_definition	has part	definition	
+ERROR	missing_label	CL:0000000	rdfs:label
```
Comment (matentzn) [inline src/ontology/reports/pato-edit.obo-obo-report.tsv:2]: that is odd... How did that happen? Can we increase the failure level to fail on ERROR @shawntanzk and check whats up with that label?

Comment (matentzn) [reply src/ontology/reports/pato-edit.obo-obo-report.tsv:2]: probably an important that was not refreshed..

**Review by matentzn — APPROVED** [review]

**Pull request merged**